contracts/RiskManagerV1.sol

```diff
@@ -226,6 +235,39 @@ contract RiskManagerV1 is Auctioneer, Ownable {
         auctionLengthChangeInitiated = 0;
     }
 
+    /// @notice Begins the signer bonds swap strategy update process.
+    /// @dev Must be followed by a finalizeSignerBondsSwapStrategyUpdate after
+    ///      the governance delay elapses.
+    /// @param _newSignerBondsSwapStrategy The new signer bonds swap strategy.
+    function beginSignerBondsSwapStrategyUpdate(
+        ISignerBondsSwapStrategy _newSignerBondsSwapStrategy
```

Comment:
Can we assert `_newSignerBondsSwapStrategy` is a non-zero address?

Reply:
Fixed in d295b8363bac9bc886cb50c33525c461ac2c3721.